aptos-move/framework/aptos-stdlib/sources/any.move

```diff
@@ -0,0 +1,61 @@
+module aptos_std::any {
+    use aptos_std::type_info;
+    use std::bcs;
+    use std::error;
+    use std::string::String;
+
+    /// The type provided for `unpack` is not the same as was given for `pack`.
+    const ETYPE_MISMATCH: u64 = 0;
+
+    /// A type which can represent a value of any type. This allows for representation of 'unknown' future
+    /// values. For example, to define a resource such that it can be later be extended without breaking
+    /// changes one can do
+    ///
+    /// ```move
+    ///   struct Resource {
+    ///      field: Type,
+    ///      ...
+    ///      extension: Option<Any>
+    ///   }
+    /// ```
+    struct Any has drop, store {
```

Comment:
I wonder if we need multiple flavors of Any with different abilities

Reply:
Maybe, but lets start with this one.